poradnia/letters/models.py

```diff
@@ -218,6 +219,13 @@ def text(self):
             return talon.quotations.extract_from(self.message.text)
         return talon.quotations.extract_from(self.message.html, 'text/html')
 
+    @cached_property
+    def html(self):
+        if self.message.html:
+            return talon.quotations.extract_from_html(self.message.html)
```

Comment:
Takie rzeczy robimy raz przy załadowaniu, a nie przy każdym odczytaniu kodu. Łapiemy sygnał zgodnie z dokumentacją http://django-mailbox.readthedocs.io/en/latest/topics/polling.html#in-your-code .

Reply:
Nie jestem pewien czy dobrze rozumiem - wydaje mi się, że w obecnej postaci ta metoda będzie wywołana tylko raz dla każdej otrzymanej wiadomości.

Reply:
Masz oczywiście racje!